### PR TITLE
WIP: Draft statement of work for comments

### PR DESCRIPTION
**Pull request by srfraser**
Comments and changes!

### CODE REVIEW

File: how-tos/releaseduty_overview.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# ReleaseDuty Statement of Work / Overview
```
Comment (MihaiTabara) [inline how-tos/releaseduty_overview.md:1]: Nit: There is a Buildduty "manifesto". I'm thinking we could keep the consistency and name this `releaseduty_manifesto` as filename. The title is very good though.

File: how-tos/releaseduty_overview.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ReleaseDuty is a duty cycle where an engineer is responsible for operational
+tasks related to releasing new versions of Firefox.  Two people are assigned to
+ReleaseDuty every six weeks, preferably in different timezones to extend
+available coverage.  There are specific days in a cycle when set tasks must be
+done, such as creating a beta or release candidate, and other unscheduled work
+that depends on issues that need fixing, and special requirements that specific
+release might have.
+
+ReleaseDuty is responsible for:
+* Performing operational tasks for incoming Nightly, DevEdition, Beta, Release Candidate and final releases
```
Comment (MihaiTabara) [inline how-tos/releaseduty_overview.md]: if you're going to enumerate them all, let's mention ESR as well.

File: how-tos/releaseduty_overview.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+* Merging repositories to create new beta, release candidates, releases and extended support releases.
+* Coordinating with other teams, such as Release Management, Quality Engineering, Sheriffs, and other developers
+* Fixing and improving the release automation, including both tools and processes
+* Maintaining a logbook of issues and progress using releasewarrior.
+* Arranging event post-mortems, if required.
+
+The individual accountable for a release is known as the 'release owner'.
```
Comment (MihaiTabara) [inline how-tos/releaseduty_overview.md]: `release_owner` in this context more or less refers to RelMan. Might be worth mentioning that here.

**Review by MihaiTabara — APPROVED** [review]

File: how-tos/releaseduty_overview.md (hunk 1 above)
Comment (srfraser) [reply how-tos/releaseduty_overview.md:1]: 'Statement of Work' has a specific meaning in project management, I'm given to understand that's what was asked for, but if it's not, we can change it.

https://en.wikipedia.org/wiki/Statement_of_work

File: how-tos/releaseduty_overview.md (hunk 3 above)
Comment (srfraser) [reply how-tos/releaseduty_overview.md]: We should mention it's a specific member of release management, as I don't think it's a group

File: how-tos/releaseduty_overview.md (hunk 1 above)
Comment (MihaiTabara) [reply how-tos/releaseduty_overview.md:1]: Yes, absolutely. I totally agree the title is good. I was just referring to renaming `releasesduty_overview` to `releaseduty_manifesto` 👍 Fine either way, though.

File: how-tos/releaseduty_overview.md (hunk 3 above)
Comment (MihaiTabara) [reply how-tos/releaseduty_overview.md]: Precisely. Sorry, that's what I meant actually, slipping a "Release Management release owner" there to make it clear it's not RelEng.

**Pull request merged**